NormalizModule.cpp

```diff
@@ -342,31 +318,29 @@ bool PyNumberToNmz(PyObject* in, long long& out)
     return true;
 }
 
-PyObject* NmzToPyNumber(long long in)
+PyObject* NmzToPyNumber(unsigned int in)
 {
-    return PyLong_FromLongLong(in);
+    return PyLong_FromUnsignedLong(in);
 }
 
-PyObject* NmzToPyNumber(libnormaliz::key_t in)
+PyObject* NmzToPyNumber(unsigned long in)
 {
-    return PyLong_FromLong(in);
+    return PyLong_FromUnsignedLong(in);
 }
 
-#ifdef ENVIRONMENT64
-PyObject* NmzToPyNumber(size_t in)
+PyObject* NmzToPyNumber(int in)
 {
     return PyLong_FromLong(in);
```

Comment:
Justification: an int always fits in a long, so this is safe.

And size_t is either `unsigned int`, or `unsigned long`, or `unsigned long long`, so by overloading for all three, we are sure to defined one version of the function which is applicable to `size_t`.